package.json

```diff
@@ -21,18 +22,18 @@
   "dependencies": {
     "collect-mentions": "^1.0.2",
     "dedent": "^0.7.0",
-    "esm": "^3.0.84",
     "mixin-deep": "^2.0.0"
   },
   "devDependencies": {
     "@tunnckocore/config": "^1.0.0",
-    "asia": "^0.19.7"
+    "asia": "^0.19.7",
+    "bili": "^3.4.2"
   },
   "files": [
     "src",
     "index.js"
   ],
-  "main": "index.js",
+  "main": "dist/parse-commit-message.cjs.js",
```

Comment:
It's not enough. All bundlers don't export "interop"-ed result. So CJS users will need to `require('...').default` which I really hate, and that's what I'm doing in the esm bridge index.js file.

Reply:
I'll take a look at this tomorrow (it's 3am my time)